neural_modelling/src/spike_source/poisson/spike_source_poisson.c

```diff
@@ -712,37 +666,37 @@ void set_spike_source_rate(uint32_t id, REAL rate) {
         } else {
             poisson_parameters[sub_id].is_fast_source = false;
             poisson_parameters[sub_id].mean_isi_ticks =
-                (uint32_t) rate * global_parameters.ticks_per_second;
+                (uint32_t) rate * params.ticks_per_second;
         }
     }
 }
 
-// Is this function actually used any more?
-void sdp_packet_callback(uint mailbox, uint port) {
+#if 0 // The code below isn't used
```

Comment:
This is used - this sets the data rates from host...

Reply:
Oh yes, it isn't, sorry!  The rates are set with multicast of course...

Reply:
Since it is dead, it is also buried.